internal/pkg/vm/internal/message/result.go

```diff
@@ -7,8 +7,20 @@ import (
 	"github.com/filecoin-project/go-filecoin/internal/pkg/encoding"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/gas"
 	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	adt_spec "github.com/filecoin-project/specs-actors/actors/util/adt"
 )
 
+// TODO this should be defined somewhere else
```

Comment:
message receipts should not have a `nil` return value, added this as a temporary solution.